app/core/download.py

```diff
@@ -270,7 +270,6 @@ def convert_dump_to_csv(df: pd.DataFrame):
     return csv_buffer
 
 
-def generate_data_dump_as_csv(app_url: str, db=Depends(get_db)):
+def generate_data_dump_as_csv(db=Depends(get_db)):
     df = get_whole_database_dump(db)
-    df = replace_slug_with_qualified_url(df, app_url)
```

Comment:
Does `replace_slug_with_qualified_url` no longer get called? in which case it can be deleted

Reply:
We'll need it again soon! For the first draft of this data dump work it's been decided not to include fully qualified URLs, but I think we are wanting them at some point. I'll pop a comment on the function.